packages/suite/src/views/dashboard/components/NewsFeed/index.tsx

```diff
@@ -133,11 +131,11 @@ const NewsFeed = ({ maxVisibleCount = 9 }: Props) => {
         <Section
             heading={<Translation id="TR_WHATS_NEW" />}
             actions={
-                <MediumLink target="_blank" rel="noopener noreferrer" href={mediumUrl}>
+                <TrezorLink target="_blank" href={mediumUrl}>
```

Comment:
```suggestion
                <TrezorLink thref={mediumUrl}>
```
TrezorLink > Link have `target="_blank"` as default

Reply:
You are totally right! 
Done, thanks!